src/cqrs/src/api/xxENSONOxx.xxSTACKSxx.Domain/MenuAggregateRoot/Aggregate/Menu.cs

```diff
@@ -177,7 +177,7 @@ public Menu(Guid id, string name, Guid tenantId, string description, bool enable
 
     public string Name { get; private set; }
 
-    public Guid TenantId { get; private set; }
```

Comment:
Is the public setter on an aggtergate needed for one of the dependencies?

Reply:
I think it is because we removed the "With" extension